tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -927,17 +933,28 @@ func triggerFullSyncInParallel(ctx context.Context, client clientset.Interface,
 	if err != nil {
 		framework.Logf("Full sync did not finish in given time, ignoring this error: %v", err)
 	}
-
-	crd := getTriggerFullSyncCrd(ctx, client, cnsOperatorClient)
+	crd, err := getTriggerFullSyncCrd(ctx, client, cnsOperatorClient)
+	if err != nil {
+		framework.Logf("couldn't get full sync crd in given time, ignoring this error: %v", err)
+	}
 	framework.Logf("INFO: full sync crd details: %v", crd)
 	updateTriggerFullSyncCrd(ctx, cnsOperatorClient, *crd)
```

Comment:
if we did not get crd above, how will this update work? IMO when we see any errors after first full sync is triggered, we can log it and return. If we see error before first full sync happens we should fail the test

Reply:
Site failure can happen in first full sync or second full sync, hence I wanted to ignore the error